pkg/deploytest/faketransport.go

```diff
@@ -14,28 +14,16 @@ import (
 	"fmt"
 	"github.com/filecoin-project/mir/pkg/events"
 	"github.com/filecoin-project/mir/pkg/pb/eventpb"
-	"strconv"
 	"sync"
```

Comment:
unsorted

Reply:
Yes, an empty line prevented gofmt from sorting it. Fixed now.